en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/L/Levitate.md

```diff
@@ -8,11 +26,11 @@ _2nd-level transmutation_
 **Components:** V, S, M (either a small leather loop or a piece of golden wire bent into a cup shape with a long shank on one end) \
 **Duration:** Concentration, up to 10 minutes
 
-Until the spell ends, a creature or object under 500 lbs. rises vertically in the air up to 20 feet and remains floating there, able to move only by pushing or pulling on fixed objects or surfaces within its reach.
+Until the spell ends, a creature or object under 500 lbs. that is either willing or that fails a Constitution saving thrown rises vertically in the air up to 20 feet and remains floating there, able to move only by pushing or pulling on fixed objects or surfaces within its reach.
 This allows the target to move as if it was climbing.
 
 On subsequent turns, you can use your action to alter the target’s altitude by up to 20 feet in either direction so long as it remains within range.
-If you have targeted yourself you may move up or down as part of your movement.
+If you have targeted yourself you may move up or down as part of your movement instead of using an action to do so.
```

Comment:
The way the A5e version was worded, it sounded like it might _still_ also cost your action to do this. The WotC version makes it clear, that that's not the case but would be difficult to substitute here. So I think this modification may be the best alternative.
(Though I'm open for suggestions of course.)